src/main/java/io/stargate/sgv2/jsonapi/api/security/ErrorChallengeSender.java

```diff
@@ -39,7 +40,7 @@ public ErrorChallengeSender(
     // create the response
     String message =
         "Role unauthorized for operation: Missing token, expecting one in the %s header."
-            .formatted(headerName);
```

Comment:
Is `headerName` argument still needed/wanted? It is injected from `stargate.auth.header-based.header-name` which would usually override default of some kind. If not, can probably remove argument.

Reply:
Want to show "Token" to users, I am not using the injected one, which is "X-Cassandra-Token"

Reply:
Mentioned in slack thread but just to cover all the bases:

Thanks Yuqi  for the reminder.  If the header is still needed (see Tatu's question above) ... then yes I think we would need to also update the Headers tab of commands in (e.g.) ...: 

https://www.postman.com/datastax/workspace/stargate-cassandra/request/25879866-2a91dfda-fc9b-4536-8b20-7dfb18d13384

Reply:
<img width="996" alt="Screenshot 2023-10-31 at 1 38 33 PM" src="https://github.com/stargate/jsonapi/assets/4041525/10d8979f-2dbc-4578-9259-36070a7846d3">

Reply:
> Want to show "Token" to users, I am not using the injected one, which is "X-Cassandra-Token"

Right, so you can then just remove `String headerName` from constructor I think.
Since it is no longer being used for anything.

Reply:
Perhaps no.
It is binded with configuration
If remove, can not even start quarkus
<img width="1894" alt="Pasted Graphic 4" src="https://github.com/stargate/jsonapi/assets/138714577/2f7c7f38-ed73-4a61-9043-06d02066efbc">

Reply:
Sorry, you can drop `headerName` but `ObjectMapper` still needs to be injected. So injection is needed but not headerName.

Reply:
understand, fixed it